src/Resources/config/services/sitemap.xml

```diff
@@ -3,13 +3,42 @@
     <services>
         <defaults autowire="false" autoconfigure="false" public="false" />
```

Comment:
This is default in Symfony and can therefore be removed

Reply:
Will fix this in a separate PR, code is unchanged here.

Reply:
Will solve in #102